storage-proofs/src/drgporep.rs

```diff
@@ -80,18 +80,23 @@ where
     }
 }
 
-impl<H, G> ParameterSetIdentifier for PublicParams<H, G>
+impl<H, G> ParameterSetMetadata for PublicParams<H, G>
 where
     H: Hasher,
-    G: Graph<H> + ParameterSetIdentifier,
+    G: Graph<H> + ParameterSetMetadata,
 {
-    fn parameter_set_identifier(&self) -> String {
+    fn identifier(&self) -> String {
         format!(
             "drgporep::PublicParams{{graph: {}; sloth_iter: {}}}",
-            self.graph.parameter_set_identifier(),
+            self.graph.identifier(),
             self.sloth_iter
         )
     }
+
+    fn sector_size(&self) -> Option<u64> {
+        // TODO: Expose sector size, if available.
```

Comment:
in which case is this needed?

Reply:
@dignifiedquire 

> in which case is this needed?

Sector size is required if we wish to publish Groth parameters for the circuit. If we don't, we can leave the return value `None`. I am not sure which PoSt construction we're going to ultimately publish, so I went through the effort of returning a non-`None` value for only VDF PoSt. 

> what about all the TODOs?

VDF PoSt is the only PoSt constructor which is used by the go-filecoin node today. Same thing goes for ZigZag PoRep. If you'd like me to go through the effort of figuring out how to expose sector size for circuits for which we aren't going to publish parameters, I will - but I don't think it's a great use of time.